.github/workflows/ghcr.yml

```diff
@@ -21,6 +21,8 @@ on:
 jobs:
   ghcr_build_and_push:
     runs-on: ubuntu-latest
+    outputs:
+      image_path: ${{ steps.build.outputs.image_path }}
```

Comment:
This label only points to `- name: Build and push ${{ matrix.image }}` which only runs on non-forks.

Reply:
Tbh I'd prefer we split this `ghcr_build_and_push` to two actions: `build` and `push`. All commits will run `build` while only main branch on opendevin repo will run `push`. Currently, forks run this `ghcr_build_and_push` even though it only does `build` but not `push`, which is counter-intuitive.

Reply:
Never mind, let me do the split in a separate PR.

Reply:
It seems harder than I expected...

Reply:
Why? I thought is will be easy. What problem you met?

Reply:
it's a multi-platform image. We cannot build it on one GitHub Action and load it in another GitHub Action